test/test_utils.py

```diff
@@ -168,6 +169,13 @@ def test_draw_invalid_boxes():
         utils.draw_bounding_boxes(img_correct, boxes, colors=colors_wrong)
 
 
+def test_draw_boxes_warning():
+    img = torch.full((3, 100, 100), 255, dtype=torch.uint8)
+
+    with pytest.warns(UserWarning, match=re.escape("Argument 'font_size' will be unused as 'font' is not set.")):
+        utils.draw_bounding_boxes(img, boxes, fill=True, font_size=11)
```

Comment:
Maybe I'm missing something here, but I don't see `boxes` defined anywhere. Is this test actually passing for you locally? You can run it with `pytest test/test_utils.py::test_draw_boxes_warning`.

Reply:
Plus, I don't think we need the `fill` argument here:

```suggestion
        utils.draw_bounding_boxes(img, boxes, font_size=11)
```

Reply:
Yep. The test passes locally. `boxes` is defined in the global scope at the top of the file.